### ISSUE TITLE
Refactor|Rename adds an extra blank line to the end of the module.

### ISSUE THREAD

**Issue by daFreeMan** [rubberduck-vba/Rubberduck#4345]
Win10, Excel2016(desktop), RD .3239

Every time `Refactor|Rename` is called an extra blank line is appended to the end of the code module. It's not critical by any means, but it can get annoying...

![2018-08-31 13_51_55-microsoft visual basic for applications](https://user-images.githubusercontent.com/11889733/44928632-6b93a200-ad26-11e8-8e27-8637bd1f9b76.png)

**Comment by comintern** [comment]
I can't replicate this in the current build (and I don't think that anything relevant was changed). Are you sure it's Rename -> Refactor?

**Comment by daFreeMan** [comment]
Yes, @comintern, I'm 100% certain, but I just discovered a simple MCVE won't cover it. 

I opened an Excel project and it showed me a very small basic module - 1 method, about a dozen lines of code. I could _not_ reproduce it. I was just about to close this out when a thought hit.

I opened a much larger basic module - 8 methods, about 400 LOC. In _this_ module, I can reproduce it at will. Here's a screen shot showing where the last line in the module was after each rename - I added a comment showing the current name of the variable I was renaming.
![2018-09-27 07_44_53-microsoft visual basic for applications - satisfaction summary master v3 23 - de](https://user-images.githubusercontent.com/11889733/46143877-5cbae500-c229-11e8-9898-4c6cc7fbd2e6.png)
After each rename, I hit `Ctrl-End` and started typing at whatever line was now at the end of the code.

Here's what it looks like after a rename cycle through `foo`, `bar`, `biff`, `baz`, and back to `sqlStatement` without typing anything additional at the end of the code in between:
![2018-09-27 07_47_46-microsoft visual basic for applications - satisfaction summary master v3 23 - de](https://user-images.githubusercontent.com/11889733/46143992-be7b4f00-c229-11e8-84a8-89e6413cc8a8.png)

_Yes, I have a method commented out at the end of the module. No I shouldn't. I don't have my code in Git yet, so this is the <s>only</s> most convenient way of holding on to some dead code that will be needed the moment I pitch it._

**Comment by IvenBach** [comment]
#5342 As it's related.
